### PR TITLE
Fix ASCN namespace import

### PR DESCRIPTION
**Pull request by pvannierop**
# Problem
Namespace import of ASCN columns from MAF file was broken in previous commit because the namespace match is now case sensitive.

# Solution
Correct case for namespace name used to decide whether to import ASCN columns.

### CODE REVIEW

**Review by onursumer — APPROVED** [review]
Would be nice to have some tests to prevent these types of regression

**Comment by pvannierop** [comment]
@onursumer I agree. But I think this is out of scope for this PR. I was not involved in implementation of the ASCN feature. Perhaps we can make an issue? However, it is good to merge this PR asap since it is resulting in errors in frontend e2e-localdb tests.

File: core/src/main/java/org/mskcc/cbio/portal/scripts/ImportExtendedMutationData.java [hunk 1]
```diff
@@ -73,7 +73,7 @@
     private Set<String> filteredMutations = new HashSet<String>();
     private Set<String> namespaces = new HashSet<String>();
     private Pattern SEQUENCE_SAMPLES_REGEX = Pattern.compile("^.*sequenced_samples:(.*)$");
-    private final String ASCN_NAMESPACE = "ascn";
+    private final String ASCN_NAMESPACE = "ASCN";
```
Comment (alisman) [inline core/src/main/java/org/mskcc/cbio/portal/scripts/ImportExtendedMutationData.java:76]: @averyniceday can see any problem with this viz-a-viz legacy data?  i.e. it was working before with lowercasing, now it won't?

Comment (pvannierop) [reply core/src/main/java/org/mskcc/cbio/portal/scripts/ImportExtendedMutationData.java:76]: No, not really. The manual describes that ASCN should be represented with capitals in the meta_mutations_extended.txt file. The lowercase conversion happens in the backend and ASCN_NAMESPACE matched this converted value. In a previous PR I have removed this conversion to lowercase. This fix will correct the match to the original case present in the meta_mutations_extended.txt file.

TLDR Legacy data is only affected when using lowecase _ascn_ in the metafile. This is not as described in the documentation and I consider this as making use a unsupported feature.
